portfolio/components/ui/Figures/SyntheticReceiptImages/SyntheticReceiptImages.module.css

```
@@ -0,0 +1,222 @@
+  font-size: 1.05rem;
+  line-height: 1.2;
+  color: var(--text-color);
+}
+
+.operation {
+  align-self: flex-start;
+  padding: 0.25rem 0.45rem;
+  border-radius: 999px;
+  background: rgba(var(--text-color-rgb, 0, 0, 0), 0.07);
+  color: var(--text-color);
+  font-family: "Monaco", "Menlo", "Consolas", monospace;
```

Stylelint (17.14.0)

[error] Expected no quotes around "Monaco" (font-family-name-quotes)

(font-family-name-quotes)